.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Help",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceRoot}/revel",
+            "args": []
+        },
+        {
+            "name": "Create new",
+            "type": "go",
+            "request": "launch",
+            "preLaunchTask": "Clean",
+            "mode": "auto",
+            "program": "${workspaceRoot}/revel",
+            "args": ["new", "-a", "/tmp/revel/aaa"],
+            "env": {
+                "GOPATH": "/tmp/revel/GOPATH"
+              },            
+        },
+        {
+            "name": "Run program",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceRoot}/revel",
+            "args": ["run","-v", "-a", "/tmp/revel/aaa"],
+            "env": {
+                "GOPATH": "/tmp/revel/GOPATH"
```

Comment:
is there a better, relative path we can use that will apply to all developer environments?

Reply:
In Linux the/tmp folder is universal not sure how we could do this to work in windows and Linux

Reply:
shouldn't it be based on go module path though?